external-resolver/src/main/scala/org/genivi/sota/resolver/vehicles/VehicleRepository.scala

```diff
@@ -71,19 +70,21 @@ object VehicleRepository {
    */
 
   // scalastyle:off
-  class InstalledFirmwareTable(tag: Tag) extends Table[(Firmware, Vehicle.Vin)](tag, "Firmware") {
+  class InstalledFirmwareTable(tag: Tag) extends
+      Table[(Firmware, Vehicle.Vin)](tag, "Firmware") {
 
     def namespace     = column[Namespace]           ("namespace")
     def module        = column[Firmware.Module]     ("module")
     def firmware_id   = column[Firmware.FirmwareId] ("firmware_id")
-    def last_modified = column[DateTime]            ("last_modified")
+    def last_modified = column[Long]                ("last_modified")
```

Comment:
Yeah, the GENIVI specification sets the format of the InstalledFirmware object and it uses longs.